keps/sig-apps/20190412-pdb-for-scale-subresource.md

```diff
@@ -0,0 +1,86 @@
+---
+title: support-scale-subresource-in-pdbs
```

Comment:
Nit: Maybe it's more clear to say: "PDB supports custom resource with scale subresource enabled", instead of "PDB supports scale subresource" (which sounds like allowing users to scale a PDB).

Reply:
I agree the title was a little unclear. I've updated it.